ui/app/src/App.tsx

```diff
@@ -33,6 +33,7 @@ function App() {
         sx={{
           paddingBottom: (theme) => theme.spacing(1),
           flex: 1,
+          display: 'flex',
```

Comment:
Some css adjustments to ensure that middle chunk of the app fills the space, so we can vertically center the empty state.